lib/mix/tasks/appsignal.install.ex

```diff
@@ -152,26 +154,36 @@ defmodule Mix.Tasks.Appsignal.Install do
 
   # Contents for the config/appsignal.exs file.
   defp appsignal_config_file_contents(config) do
+    options = [
+      ~s(  name: "#{config[:name]}"),
+      ~s(  push_api_key: "#{config[:push_api_key]}"),
+      ~s(  env: Mix.env)
+    ]
+
+    options_with_active = case has_environment_configuration_files?() do
+      false -> [~s(  active: true)] ++ options
+      true -> options
+    end
+
     "use Mix.Config\n\n" <>
       "config :appsignal, :config,\n" <>
-      ~s(  active: true,\n) <>
-      ~s(  name: "#{config[:name]}",\n) <>
-      ~s(  push_api_key: "#{config[:push_api_key]}",\n) <>
-      ~s(  env: Mix.env\n)
+      Enum.join(options_with_active, ",\n") <>
+      "\n"
   end
 
-  # Append a line to Mix configuration environment files which deactivates
-  # AppSignal for the test environment.
-  defp deactivate_config_in_test_env do
-    env_file = Path.join("config", "test.exs")
+  # Append a line to Mix configuration environment files which activate
+  # AppSignal. This is done for development, staging and production
+  # environments if they are present.
+  defp activate_config_for_env(env) do
+    env_file = config_path_for_env(env)
     if File.exists? env_file do
-      IO.write "Deactivating AppSignal in the test environment: "
+      IO.write "Activating #{env} environment: "
 
-      deactivation = "\nconfig :appsignal, :config, active: false\n"
-      case file_contains?(env_file, deactivation) do
-        :ok -> IO.puts "Success! (Already deactivated)"
+      active_content = "\nconfig :appsignal, :config, active: true, env: :#{env}\n"
```

Comment:
this sets the env in both the `config/appsignal.exs` file with `env: Mix.env` from PR #203 and in the specific env config `config/dev.exs`. I think we can remove it here?